treeherder/etl/management/commands/pulse_listener.py

```diff
@@ -17,6 +17,9 @@ class Command(BaseCommand):
     help = "Read tasks and pushes from a set of pulse exchanges and queue for ingestion"
 
     def handle(self, *args, **options):
+        if env.bool('SKIP_INGESTION', default=False):
```

Comment:
This is starting to get tricky to test.

Your earlier approach (defining SKIP_INGESTION in settings.py) seems more reasonable and easier to test (centralized configuration).

My apologies to ask to go back but would you be mind to change the usage of `env.bool` in this file to using `SKIP_INGESTION` from `settings.py`? This would mean not using `entrypoint.sh`.

Reply:
No problem @armenzg Thanks for clearing, I will make the changes and will push it.

Reply:
@armenzg I need some help, how exactly should I change env.bool in this file?

Reply:
The changes in this fine are good. I will make the comments above.